system_tests/staker_test.go

```diff
@@ -1,10 +1,6 @@
 // Copyright 2021-2022, Offchain Labs, Inc.
 // For license information, see https://github.com/nitro/blob/master/LICENSE
 
-// race detection makes things slow and miss timeouts
-//go:build !race
-// +build !race
-
```

Comment:
I think we should keep this test disabled during race detection because I'm worried it'll randomly fail due to timeouts otherwise (I think that's why we disabled it).